doc/src/vpr/debug_aids.rst

```diff
@@ -24,3 +24,82 @@ Finally, if you need to route an FPGA whose routing architecture cannot be descr
+
+Enabling/Disabling a breakpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enabling and disabling breakpoints are done using the checkboxes in front of each breakpoint in the breakpoint list. The breakpoint is enabled when the box is checked and disabled otherwise.
+
+Deleting a breakpoint
+~~~~~~~~~~~~~~~~~~~~~
+
+Deleting a breakpoint is done using the trash can button in front of each breakpoint in the breakpoint list.
+
+Reaching a breakpoint
+~~~~~~~~~~~~~~~~~~~~~
+
+Upon reaching a breakpoint, the program will stop, notify the user which breakpoint was encountered, and give a summary of the current variable values. This information is presented through a pop-up window and printed to the terminal as well.
+
+Available Variables
+~~~~~~~~~~~~~~~~~~~
+
+.. image:: https://github.com/verilog-to-routing/verilog-to-routing.github.io/blob/master/img/advancedWindow.png
+    :align: center
+
+You can also find the variables’ list in the Advanced Settings Window, on the left.
+
+**Placer Variables**
+
+  * move_num: every placer perturbation counts as a move, so the user can stop the program after a certain number of moves. This breakpoint can be enabled through the entry field on the main debugger window or using an expression. It should be noted however, that using the entry field would proceed the specified number of moves. (as in the second example)
+          * Ex. move _num == 33
+          * Ex. move_num += 4
+  * temp_count: every time the temperature is updated it counts as an increase to temp_count. This breakpoint can be enabled through the entry field on the main debugger window or using an expression. It should be noted however, that using the entry field would proceed the specified number of temperatures. (as in the second example)
+          * Ex. temp_count == 5
+          * Ex. temp_count += 5
+  * from_block:  in every placer move one or more blocks are relocated. from_block specifies the first block that is relocated in every move; and a breakpoint of this type stops the program when the first block moved is the one indicated by the user. This breakpoint can be enabled through the entry field on the main debugger window or using an expression.
+          * Ex. from_block == 83
+  * in_blocks_affected: this variable allows you to stop after your specified block was moved. Unlike "from_block" which only checks the first block relocated in every move, in_blocks_affected looks through all the blocks whose locations were changed by that move. This breakpoint can only be enabled through the use of an expression.
+          * Ex. in_blocks_affected == 1
```

Comment:
I think changing the example here to use block 83 again would help; 1 means logical true which makes this example a bit ambiguous.

Is this set as 
in_blocks_affected == 83   
if I want to check if block 83 is moved?  (I think this is the case).

Or is it set as 
in_block_affected[83] == 1 ?
(I don't think this is the case, but again changing to block 83 would help).

Reply:
ok, great I made the change. And it is the first version you used.